flask_praetorian/base.py

```diff
@@ -195,15 +252,24 @@ def _verify_password(self, raw_password, hashed_password):
         )
         return self.pwd_ctx.verify(raw_password, hashed_password)
 
+    @deprecated('Use `hash_password` instead.')
     def encrypt_password(self, raw_password):
         """
         Encrypts a plaintext password using the stored passlib password context
+
+        *NOTE* This should be deprecated as its an incorrect definition for
+            what is actually being done -- we are hashing, not encrypting
         """
         PraetorianError.require_condition(
             self.pwd_ctx is not None,
             "Praetorian must be initialized before this method is available",
         )
-        return self.pwd_ctx.encrypt(raw_password, scheme=self.hash_scheme)
+        """
+        `scheme` is now set with self.pwd_ctx.update(default=scheme) due
+            to the depreciation in upcoming passlib 2.0.
+         zillions of warnings suck.
+        """
+        return self.pwd_ctx.hash(raw_password)
```

Comment:
Instead of duplicating the logic in this deprecated function, we should just have it call `hash_password`

Reply:
Replaced

`    @deprecated('Use `hash_password` instead.')
    def encrypt_password(self, raw_password):
        """
        *NOTE* This should be deprecated as its an incorrect definition for
            what is actually being done -- we are hashing, not encrypting
        """
        return self.hash_password(raw_password)`